tasks.py

```diff
@@ -174,7 +179,7 @@ def get_iter(self, split, tokenizer, batch_size=16, shuffle=False, random_state=
             task_iters = [task.get_iter(split, tokenizer, 1, shuffle, random_state, max_length) for task in self.tasks]
             self._task_sampler_iter = MixedTaskSamplerIter(task_iters, batch_size, self.method, self.custom_task_ratio)
         else:
-            task_iters = [task.get_iter(split, tokenizer, batch_size, shuffle, random_state) for task in self.tasks]
+            task_iters = [task.get_iter(split, tokenizer, batch_size, shuffle, random_state, max_length) for task in self.tasks]
```

Comment:
I think here I intentionally didn't pass the max length because otherwise it would be the same for all tasks
For offense eval I found in the literature that the winner of the challange used 64 but most of our other datasets use 32
without passing max_length the default of each task is chosen